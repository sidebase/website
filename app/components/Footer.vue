<script setup lang="ts">
import Logo from '~/assets/branding/SideStream_logo.svg'
</script>

<template>
  <div class="w-full flex items-center justify-between lg:container mx-auto">
    <a class="w-[250px] h-[100px] overflow-hidden flex items-center justify-center" href="https://sidestream.tech" target="_blank">
      <Logo class="fill-gray-200" width="200px" height="50px" />
    </a>
    <div class="flex items-center space-x-5">
      <a href="https://www.sidestream.tech/impressum" target="_blank" class="cursor-pointer text-SideBase-green hover:text-white transition-colors text-lg font-light">
        Imprint
      </a>
      <a class="cursor-pointer text-SideBase-green hover:text-white transition-colors text-lg font-light">
        <i class="pi pi-github pt-0.5" style="font-size: 1.25rem" />
      </a>
      <a class="cursor-pointer text-SideBase-green hover:text-white transition-colors text-lg font-light">
        <i class="pi pi-twitter pt-1" style="font-size: 1.25rem" />
      </a>
    </div>
  </div>
</template>
