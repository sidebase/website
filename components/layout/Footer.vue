<script lang="ts" setup>
import SidebaseLogoDark from '~/assets/sidebase_logo_dark_just_icon_for_dark_bg.svg'
</script>

<template>
  <footer class="bg-pickled-bluewood-800 p-4 md:p-8 lg:p-10">
    <div class="mx-auto max-w-screen-xl text-center">
      <a href="https://github.com/sidestream-tech/sidebase" target="_blank" class="flex items-center justify-center text-2xl font-semibold text-white">
        <SidebaseLogoDark class="mr-3 h-8 sm:h-12" alt="sidebase logo" />
      </a>
      <p class="my-6 text-gray-400">
        A modern, best-practice, batteries-included fullstack-app starter based on Nuxt3 and TypeScript.
      </p>
      <ul class="mb-6 flex flex-wrap items-center justify-center">
        <li>
          <a href="https://www.sidestream.tech/en/imprint" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6 ">Imprint</a>
        </li>
        <li>
          <a href="https://github.com/sidebase/sidebase" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6">Github</a>
        </li>
        <li>
          <a href="https://twitter.com/sidebase_io" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6 ">Twitter</a>
        </li>
        <li>
          <a href="https://discord.gg/VzABbVsqAc" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6 ">Discord</a>
        </li>
        <li>
          <a href="https://www.sidestream.tech/en/home" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6">Sidestream</a>
        </li>
        <li>
          <a href="https://sidestream.jobs.personio.de/" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6">Jobs</a>
        </li>
        <li>
          <a href="https://www.sidestream.tech/en/contact" target="_blank" class="hover:text-sidebase-green mr-4 text-gray-400 hover:underline md:mr-6">Contact</a>
        </li>
      </ul>
      <span class="hover:text-sidebase-green text-sm text-gray-400 sm:text-center">
        Developed 2022 by
        <a href="https://www.sidestream.tech/en/home" target="_blank" class="hover:text-sidebase-green text-gray-400 underline">SIDESTREAM</a>.
      </span>
    </div>
  </footer>
</template>
