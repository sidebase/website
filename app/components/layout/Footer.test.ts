import { describe, expect, test } from 'vitest'
import Footer from './Footer.vue'
import { render } from '~/tests/testingLibraryVue'

describe('Footer', () => {
  test('mounts and is still the same', () => {
    const { html } = render(Footer)
    expect(html()).toMatchSnapshot()
  })
})
