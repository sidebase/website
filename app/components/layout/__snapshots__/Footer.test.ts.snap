// Vitest Snapshot v1

exports[`Footer > mounts and is still the same 1`] = `
"<div class=\\"w-full flex items-center justify-between lg:container mx-auto px-4\\"><a class=\\"w-[125px] h-[50px] lg:w-[200px] lg:h-[50px] overflow-hidden flex items-center justify-center\\" href=\\"https://sidestream.tech\\" target=\\"_blank\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 686.86 46.93\\" class=\\"fill-gray-200\\">
      <defs></defs>
      <g id=\\"Ebene_2\\" data-name=\\"Ebene 2\\">
        <g id=\\"Ebene_1-2\\" data-name=\\"Ebene 1\\">
          <rect class=\\"cls-1\\" x=\\"534.49\\" y=\\"3.47\\" width=\\"58.95\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <rect class=\\"cls-2\\" x=\\"536.16\\" y=\\"35.73\\" width=\\"74.52\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <rect class=\\"cls-2\\" x=\\"599\\" y=\\"3.47\\" width=\\"58.95\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <rect class=\\"cls-1\\" x=\\"569.52\\" y=\\"19.6\\" width=\\"58.95\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <rect class=\\"cls-2\\" x=\\"634.59\\" y=\\"19.6\\" width=\\"23.36\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <rect class=\\"cls-2\\" x=\\"535.04\\" y=\\"19.6\\" width=\\"29.47\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <rect class=\\"cls-1\\" x=\\"614.01\\" y=\\"35.73\\" width=\\"43.93\\" height=\\"8.34\\" rx=\\"1.76\\"></rect>
          <circle class=\\"cls-2\\" cx=\\"667.95\\" cy=\\"7.92\\" r=\\"4.45\\"></circle>
          <circle class=\\"cls-2\\" cx=\\"682.41\\" cy=\\"7.92\\" r=\\"4.45\\"></circle>
          <circle class=\\"cls-1\\" cx=\\"669.07\\" cy=\\"23.49\\" r=\\"4.45\\"></circle>
          <path class=\\"cls-1\\" d=\\"M9 31.79A8 8 0 0 0 17.23 38c3.57 0 6.82-1.3 6.82-4.68 0-1.76-1-3.38-5.26-4.88l-4.94-1.69C3.77 23.34 2.27 17 2.27 13.26 2.27 5.07 8.71 0 17.62 0c7 0 12.48 3.38 15.21 10l-8.19 4c-1.95-4.61-5.14-5.13-7-5.13-3.12 0-5.53 1.23-5.53 4.22 0 1.17.33 3.32 4.88 4.88l4.81 1.69C32 22.88 33.8 29 33.87 33.15c0 9.56-8.32 13.78-16.32 13.78C8.84 46.93.91 42 0 33.54ZM62.33 46.22H52.26V.71h10.07ZM100.55 46.22h-17V.71h17c14.89 0 24.19 10.21 24.19 22.76s-9.3 22.75-24.19 22.75Zm0-37.51h-7v29.51h7c8.65 0 14-6.63 14-14.75s-5.35-14.76-14-14.76ZM143.13 46.22V.71h30.22v8H153.2v11.18h17.49v8H153.2v10.33h20.15v8ZM197.6 31.79a8 8 0 0 0 8.25 6.21c3.58 0 6.83-1.3 6.83-4.68 0-1.76-1-3.38-5.27-4.88l-4.94-1.69C192.4 23.34 190.9 17 190.9 13.26 190.9 5.07 197.34 0 206.24 0c7 0 12.48 3.38 15.21 10l-8.19 4c-1.95-4.61-5.13-5.13-7-5.13-3.12 0-5.52 1.23-5.52 4.22 0 1.17.32 3.32 4.87 4.88l4.81 1.69c10.21 3.18 12 9.29 12.09 13.45 0 9.56-8.32 13.78-16.31 13.78-8.71 0-16.64-4.94-17.55-13.39ZM272.74.71v8.46H259.8v37.05h-9.94v-37h-12.81V.71ZM315.12 29.32c1.17 4.81 3 8.71 7.54 8.71a7.15 7.15 0 0 0 4.61-2l3.58 7c-3 2.34-5.46 3.9-10.14 3.9-10.79 0-14.11-12.09-14.63-16.9h-6v16.19H290V.71h20.21c9.36 0 14.5 7 14.5 15 .03 6.39-3.61 11.72-9.59 13.61ZM308.94 22c3.58 0 5.72-2.6 5.72-6.37s-2.14-6.43-5.72-6.43H300V22ZM346.31 46.22V.71h30.23v8h-20.15v11.18h17.49v8h-17.49v10.33h20.15v8ZM390.38 46.22 407.93.71h9.82l17.81 45.51H425l-3.25-8.52v.13H403.9v-.13l-3.12 8.52Zm16.25-16.64h12.29l-6-18.2ZM473.45 45.7l-11.37-27.17-2.41 27.69H449.6L453.82.71H464l12.7 30.62L489.44.71h10.14l4.23 45.51h-10.08l-2.4-27.69L480 45.7Z\\"></path>
        </g>
      </g>
    </svg></a>
  <div class=\\"flex items-center space-x-5\\"><a href=\\"https://www.sidestream.tech/impressum\\" target=\\"_blank\\" class=\\"cursor-pointer text-SideBase-green hover:text-white transition-colors text-lg font-light\\"> Imprint </a><a href=\\"https://github.com/sidestream-tech/sidebase\\" target=\\"_blank\\" class=\\"cursor-pointer text-SideBase-green hover:text-white transition-colors text-lg font-light\\"><i class=\\"pi pi-github pt-0.5\\" style=\\"font-size:1.25rem;\\"></i></a><a href=\\"https://twitter.com/sidebase_io\\" target=\\"_blank\\" class=\\"cursor-pointer text-SideBase-green hover:text-white transition-colors text-lg font-light\\"><i class=\\"pi pi-twitter pt-1\\" style=\\"font-size:1.25rem;\\"></i></a></div>
</div>"
`;
